sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -29,12 +31,9 @@
  * requests.
  */
 public class OpenTelemetryTracer implements com.azure.core.util.tracing.Tracer {
-    private static final Tracer TRACER = OpenTelemetry.getTracerProvider().get("Azure-OpenTelemetry");
-
+    private static final Tracer TRACER = GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry");
     // standard attributes with AMQP request
     static final String AZ_NAMESPACE_KEY = "az.namespace";
-    static final String MESSAGE_BUS_DESTINATION = "message_bus.destination";
-    static final String PEER_ENDPOINT = "peer.address";
```

Comment:
These were updated to follow https://github.com/open-telemetry/opentelemetry-java/blob/master/semconv/src/main/java/io/opentelemetry/api/trace/attributes/SemanticAttributes.java
Should we do the same in the exporter?
message_bus.destination ->  messaging.destination

Reply:
Yes, we should use the same name in exporter too.